octobot/cli.py

```diff
@@ -232,6 +234,11 @@ def octobot_parser(parser):
                         help='On multiple files backtesting: run on the whole available data instead of the '
                              'common part only (default behavior).',
                         action='store_true')
+    parser.add_argument('-ebt', '--enable-backtesting-timeout',
+                        help='To disable when running long backtesting : disable backtesting watcher timeout '
```

Comment:
We should remove this first part of the help, there is nothing to disable as it's not enabled by default.